### PR TITLE
MMBN3: Adds instructions for using the Legacy Collection ROM for setup

### PR DESCRIPTION
**Pull request by digiholic**
## What is this fixing or adding?
Adds instructions to MMBN3's Setup Guide for how to extract the ROM from the PC version of the Legacy Collection.

## How was this tested?
Instructions provided and verified by Discord users (Notably tauakiou who provided all the file names in one place, thanks!) who have successfully patched and played the randomizer with files from the Legacy Collection

### CODE REVIEW

File: worlds/mmbn3/docs/setup_en.md [hunk 1]
```diff
@@ -35,6 +36,14 @@ To do so, we simply have to search any GBA rom we happened to own, right click a
 the list that appears and select the bottom option "Look for another application", then browse to the Bizhawk folder
 and select EmuHawk.exe.
 
+## Extracting a ROM from the Legacy Collection
+
+The Steam version of the Legacy Collection contains unmodified GBA ROMs in its files. You can extract these for use with Archipelago.
+
+1. Open the Legacy Collection Vol. 1's Game Files (Right click on the game in your Library, then open Properties -> Installed Files -> Browse)
+2. Open the file `exe/data/exe3b.dat` in a Zip extracting program such as 7Zip or WinRAR.
```
Comment (Silvris) [inline worlds/mmbn3/docs/setup_en.md:44]:
```suggestion
2. Open the file `exe/data/exe3b.dat` in a zip-extracting program such as 7-Zip or WinRAR.
```

**Review by Silvris — COMMENTED** [review]
Just one small change I'd personally make, looks good to me otherwise.

**Review by Berserker66 — APPROVED** [review]
Content wise, I followed this to get my rom file. Quality of English I don't want to judge.

**Comment by eudaimonistic** [comment]
These instructions work perfectly and are easy to follow.  The English is fine.  There was some discussion in #megaman-battle-network-3 in the community discord about including instructions for extraction using W10 and W11 native extraction tools, but I think recommending a free software such as 7zip first is actually the right decision.  If I had to make a suggestion to improve this detail, it would be to add (free) next to 7zip, but frankly I think that's overkill as most people can figure this out on their own.

**Comment by black-sliver** [comment]
The program is called "7-Zip", so I'd like to see silvris' suggestion being merged

**Pull request merged**